src/test/java/com/microsoft/azure/spring/data/cosmosdb/config/AbstractDocumentDbConfigurationIT.java

```diff
@@ -19,12 +19,14 @@
 import org.junit.rules.ExpectedException;
 import org.mockito.Mock;
 import org.springframework.beans.factory.NoSuchBeanDefinitionException;
+import org.springframework.beans.factory.annotation.Value;
 import org.springframework.context.annotation.AnnotationConfigApplicationContext;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.PropertySource;
 import org.springframework.context.support.AbstractApplicationContext;
 
-public class AbstractDocumentDbConfigurationUnitTest {
+public class AbstractDocumentDbConfigurationIT {
```

Comment:
Why turn this unit test into an integration test?

Reply:
DocumentDbTemplate bean creation requires now real uri and key, because cosmosClient gets initialized in constructor, so had to change it to integrationTest to grab the real uri and key.